tools/python/dhis2_plugins/dhis2_docs/dhis2_utils.py

```diff
@@ -123,8 +126,7 @@ def __init__(self,config,tx_slug,template):
     #     # Delete all temporary files
     #     shutil.rmtree(self.root)
 
-
-    def include_git(self, github_repo, file_path, branch, local_path):
+    def clone_git(self, github_repo, branch):
```

Comment:
I refactored this because I initially intended to use the clone to to get access to the .yml in the remote repository, but later decided to send a lightweight request to the raw github url.